docs/speaker-enrollment-api.md

```diff
@@ -224,7 +224,7 @@ curl -X GET \
 
 ### Query Parameters
 
-| Parameter      | Type   | Description | Notes                                                                       |
-| -------------- | ------ | ----------- | --------------------------------------------------------------------------- |
-| apikey         | String | The apikey  | Required for authentication inside all requests                             |
-| getAllSpeakers | Bool   | true/false  | Set to `true` for fetching incomplete enrolled speakers too. Default to `false` |
+| Parameter      | Type   | Description | Notes                                                                            |
+| -------------- | ------ | ----------- | -------------------------------------------------------------------------------- |
+| apikey         | String | The apikey. | Required for authentication inside all requests.                                 |
+| getAllSpeakers | Bool   | true/false. | Set to `true` for fetching incomplete enrolled speakers too. Default to `false`. |
```

Comment:
standardise true/false to the capitalized variant.

Reply:
done

Reply:
done